extensions/interactions/GraphInput/directives/graph-viz.component.ts

```diff
@@ -84,6 +84,8 @@ export class GraphVizComponent implements OnInit, AfterViewInit {
   @Input() canEditEdgeWeight: boolean;
   @Input() interactionIsActive: boolean;
   @Input() canEditOptions: boolean;
+  dotCoordinateX: number = 0;
```

Comment:
Not sure what "dot" refers to here. "cursor", perhaps? I think in general you might want to use "cursor" rather than "dot" for this because it connotes "the thing that blinks and indicates where on the screen a user is making their edits".

Reply:
@seanlip In the same component we use `mouseX` and `mouseY` names as well, still do you think "cursor" would not create confusion? What do you think of using the name -- "dotCursor" ?

Reply:
I think "mouse" refers to the location of the mouse pointer on the screen, and "cursor" refers to the thing on the screen that is blinking -- so those are two different things. Is my interpretation of "mouse" correct? I'm also fine with "dotCursor" if you think that's better.

Reply:
Yes that's correct, I'm thinking of the cursor as the red dot.